include/deal.II/base/parameter_acceptor.h

```diff
@@ -516,6 +520,72 @@ class ParameterAcceptor : public Subscriptor
 
 
 
+/**
+ * A proxy ParameterAcceptor wrapper for classes that have a static member
+ * function @p declare_parameters, and a non virtual @p parse_parameters method.
+ *
+ * If a class already implements the static member function declare_parameters,
+ * and the method declare_parameters, then it cannot be derived directly from
+ * ParameterAcceptor, as the existing member functions would be incompatible
+ * with the virtual methods with the same name that ParameterAcceptor expects
+ * you to overload. For these cases, you can still use ParameterAcceptor
```

Comment:
The statement "if a class...then it cannot be derived directly....". I *think* that the use case you are thinking of here is that someone has already implemented a class and is thinking how to make it compatible with `ParameterAcceptor`. So when you say "cannot be derived" you have in mind a process ("you can't rewrite this class in a way where it then derives from `ParameterAcceptor`") as opposed to a statement about the current way the class is declared ("close considerations will show that the class can not have been declared as derived from `ParameterAcceptor`; we know this even without looking at the actual declarations because if it were, this would result in a conflict and the compiler would reject the class").

Can you maybe rephrase this statement to clarify that you are talking about converting an existing class?